openerp/addons/base/migrations/9.0.1.3/pre-migration.py

```diff
@@ -49,6 +49,7 @@ def cleanup_modules(cr):
             ('contacts', 'mail'),
             ('marketing_crm', 'crm'),
             ('email_template', 'mail'),  # mail_template class
+            ('portal_claim', 'crm_claim'),
```

Comment:
Looking at the code at 8.0 and 9.0, it's better to just rename (in apriori.py) `portal_claim` to `website_crm_claim`